_installation/mac_os.md

```diff
@@ -0,0 +1,118 @@
+---
+url: mac_os.html
+number: i
+title: Installation on Mac OS X
+layout: rails_tutorial
+---
+
+{:.sectionheader}
+# Mac OS X install
+
+To get a mac ready for Ruby on Rails development, we need to do the following:
+
+1. install command-line tools (XCode)
+2. install ruby
+3. install rails
```

Comment:
I prefer numbering all ordered list items with `1` because there's less stuff to worry about if the list is edited later.

```
1. install command-line tools (XCode)
1. install ruby
1. install rails
```